src/core/SkipToContent/SkipToContent.test.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { render } from '@testing-library/react';
+import { SkipToContentLink } from './SkipToContentLink';
+import { SkipToContentMarker } from './SkipToContentMarker';
+
+it('should render link in its most basic state', () => {
+  const { container } = render(<SkipToContentLink />);
+  const link = container.querySelector(
+    '.iui-skip-to-content-link',
+  ) as HTMLElement;
+  expect(link).toBeTruthy();
+  expect(link.textContent).toBe('Skip to main content');
+  expect(link).toHaveAttribute('href', '#skip-to-content-marker');
+});
+
+it('should render marker in its most basic state', () => {
+  const { container } = render(<SkipToContentMarker />);
+  expect(container.querySelector('#skip-to-content-marker')).toBeTruthy();
+});
+
+it('should render link with children', () => {
+  const { container } = render(
+    <SkipToContentLink>Custom child text</SkipToContentLink>,
+  );
+
+  const link = container.querySelector(
+    '.iui-skip-to-content-link',
+  ) as HTMLElement;
+  expect(link).toBeTruthy();
+  expect(link.textContent).toBe('Custom child text');
+});
+
+it('should render marker with children', () => {
+  const { container } = render(
+    <SkipToContentMarker>
+      <a href='#' id='main'>
+        main link
+      </a>
+    </SkipToContentMarker>,
+  );
+
+  const marker = container.querySelector('#skip-to-content-marker');
+  expect(marker).toBeTruthy();
+  expect(marker?.children.length === 1);
+  expect(marker?.querySelector('#main')).toBeTruthy();
+});
+
+it('should propagate misc props in link (except href)', () => {
```

Comment:
This doesn't really test the `except href` part so the name looks wrong. But as I mentioned in my other comment, we should allow overriding href.

Reply:
Fixed